src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final double firstSide;
+    private final double secondSide;
+
+    public Rectangle(String color, double firstSide, double secondSide) {
+        super(color);
+
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+    }
+
+    @Override
+    public double getArea() {
+        return firstSide * secondSide;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println(
+                "Figure: rectangle, area: " + getArea() + " sq. units, first side: "
```

Comment:
Use name() for getting String representation of enum constants. Instead of concatenating the color directly, you should call color.name() to get the String representation of the enum constant.

Reply:
No, besauce the type of the color property is String.